zingolib/src/wallet/send.rs

```diff
@@ -274,7 +274,7 @@ impl LightWallet {
         let witness_trees = txmds_readlock
             .witness_trees
             .as_ref()
-            .ok_or("No spend capability.")?;
+            .ok_or_else(|| "No spend capability.".to_string())?;
```

Comment:
`?` is (almost certainly) lazily evaluating, or the `&str` ("No spend capability."), would not match the `..., String>` in the fn return type.   So we can go with the marginally more readable original version.